src/cryptonote_core/service_node_list.cpp

```diff
@@ -936,8 +927,9 @@ namespace service_nodes
     uint64_t const block_height    = cryptonote::get_block_height(block);
 
     crypto::public_key key;
-    auto info_ptr = std::make_shared<service_node_info>();
+    auto info_ptr           = std::make_shared<service_node_info>();
     service_node_info &info = *info_ptr;
+    info                    = {};
```

Comment:
This is going to end up doing a 0 initialization of a new instance, copy that into `info`, then destruct the new instance.  It might be cleaner to just default `= 0` on the fields in the header instead so that you don't have to worry about initialization at all.

Reply:
And updated